01_standalone_examples/databricks-ci-cd/README.md

```diff
@@ -0,0 +1,222 @@
+
+        lakefs_access_key_id e.g. 'AKIAIOSFOLKFSSAMPLES'
+
+        lakefs_secret_access_key e.g. 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
+
+1. Create a Git repository. It can be named **lakeFS-samples-ci-cd**.
+
+1. Clone this repository:
+
+   ```bash
+   git clone https://github.com/treeverse/lakeFS-samples && cd lakeFS-samples/01_standalone_examples/aws-databricks/ci-cd-demo
+   ```
+
+1. Create folders **.github/workflows** and **databricks-notebooks** in your Git repo.
+
+1. Upload **pr_commit_run_databricks_etl_job.yml** file in **lakeFS-samples/01_standalone_examples/aws-databricks/ci-cd-demo/.github/workflows** folder to **.github/workflows** folder in your Git repo.
+
+1. Upload all files in **lakeFS-samples/01_standalone_examples/aws-databricks/ci-cd-demo/databricks-notebooks** folder to **databricks-notebooks** folder in your Git repo.
+
+1. Add following secrets in your Git repo by following [Creating secrets for a repository docs](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository). This is the Databricks token created in 1st step above.
+
+        DATABRICKS_TOKEN
+
+
+1. Add following variables in your Git repo by following [Creating configuration variables for a repository docs](https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository):
+* Variable to store your [Databricks host name or URL](https://docs.databricks.com/en/workspace/workspace-details.html#workspace-instance-names-urls-and-ids) e.g. https://cust-success.cloud.databricks.com
+
+        DATABRICKS_HOST 
+
+* Variable to store your [Databricks Cluster ID](https://docs.databricks.com/en/workspace/workspace-details.html#cluster-url-and-id) e.g. 1115-164516-often242
+
+        DATABRICKS_CLUSTER_ID
+
+* Variable to store your [Databricks Workspace Folder path](https://docs.databricks.com/en/workspace/workspace-details.html#folder-id) e.g. /Shared/lakefs_demos/ci_cd_demo or /Users/me@example.com/MyFolder/lakefs_demos/ci_cd_demo
+
+        DATABRICKS_WORKSPACE_NOTEBOOK_PATH 
+
+* Variable to store your Databricks Secret Scope created in 2nd step e.g. demos
+
+        DATABRICKS_SECRET_SCOPE
+
+* Variable to store your lakeFS End Point e.g. https://company.region.lakefscloud.io
+
+        LAFEFS_END_POINT
+
+* Variable to store your lakeFS repository name (which will be created by this demo) e.g. databricks-ci-cd-repo
+
+        LAKFES_REPO_NAME
+
+* Variable to store the storage namespace for the lakeFS repo. It is a location in the underlying storage where data for lakeFS repository will be stored. e.g. s3://example
+
+        LAKEFS_REPO_STORAGE_NAMESPACE
+
+* Variable to store the storage namespace where Delta tables created by this demo will be stored e.g. s3://data-source/delta-tables. Do NOT use the same storage namespace as above.
+
+        DATA_SOURCE_STORAGE_NAMESPACE
+
```

Comment:
We should also add here that the databricks user needs permissions to write to the catalogs external data -> external locations